### DISCUSSION TITLE
Permissions / Forbidden

### DISCUSSION THREAD

**Discussion by asi-jake** [mawoka-myblock/ClassQuiz#381] [Q&A]
So, I think I'm having a bit of a brain fart. I know how to modify permissions, etc. I'm on Ubuntu with freshly cloned and built ClassQuiz setup. I believe all permissions are correct. I've even tried 755 and 777 to see if it makes a difference. For the life of me, I cannot figure out why I am receiving a Forbidden message when browsing to the page. There were no errors that I saw during the build. It is possible there were some and I missed them. Any thoughts on where to look for what permissions are not correct?

**Comment by mawoka-myblock** [comment]
Which page are you referring to? The file permissions won't be the problem

**Reply by asi-jake** [reply]
In general. I cannot access the site at all. I have https as well.

Site is quiz.ameriserveintl.net

**Reply by mawoka-myblock** [reply]
The issue is not related to ClassQuiz, as the error originates from Apache, which is not a part of ClassQuiz. Take a look at your reverse proxy apache config.

**Reply by asi-jake** [reply]
I didn't figure it was related to ClassQuiz and that it is related to the server setup. I was just hoping you might have some insight for this. I have multiple VMs on GCP. The VM I have this set up on currently runs two other sites without issue. I don't have any specific forward or reverse proxy set up and have a static IP for this server and ClassQuiz set up with its own VirtualHost file. Am I supposed to set up a reverse proxy?

**Reply by mawoka-myblock** [reply]
As you can see here (https://github.com/mawoka-myblock/ClassQuiz/blob/a686aca841a7d680bbd38d697a3258f5058c73e6/docker-compose.yml#L103), the port is set normally to 8080. Point your reverse proxy to that or adapt the `Caddyfile-docker` and the port mappings around that Line 103.

**Reply by asi-jake** [reply]
Apparently, I missed that when I was going through the file. Just enabling proxy and adding to the virtual host resolved the forbidden issue. Now I'm receiving:

Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

**Reply by mawoka-myblock** [reply]
That still seems to be an Apache issue
